scripts/check_enclave_reproducibility

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+# This script builds the Asylo server twice from scratch (cleaning the Bazel cache before each
+# compilation), saves the hash of the trusted enclave code to a temporary file, and compares the
+# hashes of the two artifacts.
+# It prints nothing if the hashes match, or a diff in case they don't match.
+
```

Comment:
Maybe leave a TODO as a reminder that there are [lots of things to try varying](http://manpages.ubuntu.com/manpages/cosmic/man1/reprotest.1.html#variations) when testing reproducibility

Reply:
Interesting, thanks!